PQAnalysis/traj/trajectory.py

```diff
@@ -143,6 +143,77 @@ def __iter__(self) -> Iterable[AtomicSystem]:
             frame.topology = self.topology
             yield frame
 
+    def window(self, step: int, gap: int = 1, start: int = 0, stop: int = 0) -> Iterable[AtomicSystem]:
```

Comment:
Shouldn't this be `Iterable[List[AtomicSystem]]`

```suggestion
    def window(self, step: int, gap: int = 1, start: int = 0, stop: int = 0) -> Iterable[List[AtomicSystem]]:
```
or maybe to avoid confusions

```suggestion
    def window(self, step: int, gap: int = 1, start: int = 0, stop: int = 0) -> Generator[List[AtomicSystem]]:
```